### PR TITLE
[DOCFIX] improve docs/en/Running-Presto-with-Alluxio.md

### PR DESCRIPTION
**Pull request by LuQQiu**
make the presto cli example easier to follow.

### CODE REVIEW

File: docs/en/Running-Presto-with-Alluxio.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ Presto's Hive integration uses the config [`hive.max-split-size`](https://terada
 We recommend you to download the tarball from
 Alluxio [download page](http://www.alluxio.org/download).
 Alternatively, advanced users can choose to compile this client jar from the source code
-by following Follow the instructs [here](Building-Alluxio-From-Source.html#compute-framework-support).
+by following the instructs [here](Building-Alluxio-From-Source.html#compute-framework-support).
```
Comment (gpang) [inline docs/en/Running-Presto-with-Alluxio.md]: "instructs" -> "instructions"

File: docs/en/Running-Presto-with-Alluxio.md [hunk 2]
```diff
@@ -137,9 +139,33 @@ View Alluxio WebUI at `http://master_hostname:19999` and you can see the directo
...
-/home/path/presto/presto-cli-0.191-executable.jar --server masterIp:prestoPort --execute "use default;select * from u_user limit 10;" --user username --debug
+
+Start your Presto server. Presto server runs on port 8080 by default:
+
+```bash
+$ /<PATH_TO_PRESTO>/bin/launcher run
+```
+
+Download the `presto-cli-<YOUR_PRESTO_VERSION>-executable.jar` from [Presto CLI guidence](https://prestodb.io/docs/current/installation/cli.html) 
+and rename it to presto.
```
Comment (gpang) [inline docs/en/Running-Presto-with-Alluxio.md]: why do you have to rename it to presto? Where do you have to save this file?

Comment (LuQQiu) [reply docs/en/Running-Presto-with-Alluxio.md]: Sometimes the ./bin/presto is directly installed, like brew install presto. The executable jar is already downloaded, renamed and made executable.

directly using an executable.jar make me feel confusing at the beginning until I saw the official docs.

Rename it to presto is to sync with Presto official docs and make new users like me who already have the renamed presto executable file inside the installation folder easier to figure out the presto file could be used.

changed to ./presto

File: docs/en/Running-Presto-with-Alluxio.md [hunk 3]
```diff
@@ -137,9 +139,33 @@ View Alluxio WebUI at `http://master_hostname:19999` and you can see the directo
...
-/home/path/presto/presto-cli-0.191-executable.jar --server masterIp:prestoPort --execute "use default;select * from u_user limit 10;" --user username --debug
+
+Start your Presto server. Presto server runs on port 8080 by default:
+
+```bash
+$ /<PATH_TO_PRESTO>/bin/launcher run
+```
+
+Follow [Presto CLI guidence](https://prestodb.io/docs/current/installation/cli.html) to download to `presto-cli-<PRESTO_VERSION>-executable.jar`,
+rename it to presto, make it executable with `chmod +x`.
```
Comment (gpang) [inline docs/en/Running-Presto-with-Alluxio.md]: Please use backticks with `presto`.

Does this file have any relationship with `/<PATH_TO_PRESTO>/bin/`?

Renaming a `.jar` file to an executable just works?

**Comment by gpang** [comment]
@LuQQiu Thanks for these fixes! Does this have to be cherry-picked to `branch-1.8`?

**Pull request merged**